sphinx_gallery/gen_rst.py

```diff
@@ -313,8 +313,12 @@ def scale_image(in_fname, out_fname, max_width, max_height):
     width_sc = int(round(scale * width_in))
     height_sc = int(round(scale * height_in))
 
-    # resize the image
-    img.thumbnail((width_sc, height_sc), Image.ANTIALIAS)
+    # resize the image using resize; if using .thumbnail and the image is
+    # already smaller than max_width, max_height, then this won't scale up
+    # at all (maybe could be an option someday...)
+    img = img.resize((width_sc, height_sc), Image.BICUBIC)
+    # img.thumbnail((width_sc, height_sc), Image.BICUBIC)
+    # width_sc, height_sc = img.size  # necessary if using thumbnail
```

Comment:
Using the thumbnail (non-upsampling) method if we ever want it (as suggested by the preceding comments)